fastlane/Fastfile

```diff
@@ -6,7 +6,7 @@ platform :ios do
     run_tests(
       scheme: "Ably-iOS-Tests",
       devices: ["iPhone XS (12.0)"],
-      xcargs: { ABLY_ENV: ENV['ABLY_ENV'] }
+      xcargs: { ABLY_ENV: ENV['ABLY_ENV'], CLANG_ANALYZER_OUTPUT: 'plist-html', CLANG_ANALYZER_OUTPUT_DIR: '$(pwd)/clang' }
```

Comment:
This pattern is quite repetitive. Could we DRY this up? I also worry that the apparent, explicit choice of `clang` as the path segment name could overload / overlap with other similar folders. Perhaps a longer name like `clang-analyzer-output` would make this clearer - both here and in the upstream (workflow-located) `find` statement?

Reply:
d048bdfc405458524b48de4221e7cde066665947